app/backend/src/database/repositories/BlockValueRepository.ts

```diff
@@ -59,8 +60,21 @@ export class BlockValueRepository {
     records: BlockValueRecord[],
   ): Promise<void> {
     const tableName = this.getTable(chainId);
+    const serializedRecords = records.map((record) => ({
+      ...record,
+      value: JSON.stringify({
+        ...record.value,
+        unmapped: record.value.unmapped.map((item) => ({
+          ...item,
+          rawTotalAmount: item.rawTotalAmount.toString(),
+        })),
+      }),
+      gas_fees: record.gas_fees.toString(),
```

Comment:
It would be harder to do a `sum` in database if all these values will be strings. Postgres `bigint` should be enough, alternatively there is a `numeric` type with configurable limits.

Reply:
Was taling about `gas_fees` and `block_reward` but I see they also have `number` USD equivalents.

Also, I do support the removal of unmapped assets :)